safeguard/Readme.md

```diff
@@ -6,4 +6,6 @@ Kafka is an extremely configurable system. This flexibility is great but can lea
 
 * [Create topic Safeguard](createTopic/Readme.md) - Limits on topic creation to ensure that any topics created in the cluster adhere to a minimum specification for Replication Factor and Partition count.
 * [Alter configs Safeguard](alterConfigs/Readme.md) - Protect Kafka from inefficient configurations.
-* [On produce Safeguard](onProduce/Readme.md) - Protect Kafka from inefficient messages.
\ No newline at end of file
+* [On produce Safeguard](onProduce/Readme.md) - Protect Kafka from inefficient messages.
+* [Limit Clients Calls](limitClient/Readme.md) - Limit client calls to Kafka.
+* [Limit Clients Connections](limitConnect/Readme.md) - Limit client connections.
```

Comment:
Limit connections to the kafka cluster, or to the proxy?

Reply:
fixed